SharedPackages/BrowserServicesKit/Sources/BrowserServicesKit/PrivacyConfig/Features/PrivacyFeature.swift

```diff
@@ -131,12 +131,6 @@ public enum AIChatSubfeature: String, Equatable, PrivacySubfeature {
         .aiChat
     }
 
-    /// Displays the settings item for showing a shortcut in the macOS application menu.
```

Comment:
Following with the [removal of FF](https://app.asana.com/0/1200194497630846/1209313616180294/f) approach. I'm removing some old AI Chat feature flags here